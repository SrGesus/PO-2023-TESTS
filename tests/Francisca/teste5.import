linhas=41
colunas=32
1;1|5
1;2|49
1;3|1
1;4|'abc
1;5|'7
2;1|25
2;2|43
2;3|5
3;1|10
3;2|5
3;3|4
4;1|3
4;2|1
4;3|2
4;4|2
4;5|2
4;6|
4;7|
4;9|
4;8|