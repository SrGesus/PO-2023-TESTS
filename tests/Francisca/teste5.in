4
3
1;1:1;3
7
1
4;6:4;9
6
4;6:4;9
1
4;6:4;9
0
0