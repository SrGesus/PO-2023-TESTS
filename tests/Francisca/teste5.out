Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: 1;1|5
1;2|49
1;3|1
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;6|
4;7|
4;8|
4;9|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Especifique a gama (startline;startcol:endline;endcol): 4;6|
4;7|
4;8|
4;9|
Menu de Edição
1 - Visualizar
2 - Inserir
3 - Copiar
4 - Apagar
5 - Cortar
6 - Colar
7 - Mostrar 'cut buffer'
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Menu de Edição
5 - Menu de Consultas
0 - Sair
Escolha uma opção: 